pkg/controller/migration/objects_test.go

```diff
@@ -37,3 +41,158 @@ func Test_configMapNameForMigration(t *testing.T) {
+				ObjectMeta: v1.ObjectMeta{Name: "my-table"},
+				Spec: schemasv1alpha3.TableSpec{
+					Database: "my-database",
+					Name:     "my-table",
+					Schema: &schemasv1alpha3.TableSchema{
+						Postgres: &schemasv1alpha3.SQLTableSchema{
+							PrimaryKey: []string{"id"},
+							Columns: []*schemasv1alpha3.SQLTableColumn{
+								{
+									Name: "id",
+									Type: "text",
+									Constraints: &schemasv1alpha3.SQLTableColumnConstraints{
+										NotNull: new(bool),
+									},
+								},
+								{
+									Name: "name",
+									Type: "text",
+									Constraints: &schemasv1alpha3.SQLTableColumnConstraints{
+										NotNull: new(bool),
+									},
+								},
+							},
+						},
+					},
+				},
+				Status: schemasv1alpha3.TableStatus{},
+			},
+		},
+		{
+			name:                "Configures correctly when not using vault",
+			expectedAnnotations: nil,
+			expectedArgs: []string{
+				"plan",
+				"--driver",
+				"postgres",
+				"--spec-file",
+				"/specs/table.yaml",
+				"--uri",
+				"postgres://user:password@postgres:5432/my-database",
+			},
+			database: &databasesv1alpha3.Database{
+				TypeMeta:   v1.TypeMeta{APIVersion: "databases.schemahero.io/v1alpha3", Kind: "Database"},
+				ObjectMeta: v1.ObjectMeta{Name: "my-database"},
+				Spec: databasesv1alpha3.DatabaseSpec{
+					Connection: databasesv1alpha3.DatabaseConnection{
+						Postgres: &databasesv1alpha3.PostgresConnection{
+							URI: databasesv1alpha3.ValueOrValueFrom{
+								Value: "postgres://user:password@postgres:5432/my-database",
+							},
+						},
+					},
+				},
+				Status: databasesv1alpha3.DatabaseStatus{},
+			},
+			table: &schemasv1alpha3.Table{
+				TypeMeta:   v1.TypeMeta{APIVersion: "schemas.schemahero.io/v1alpha3", Kind: "Table"},
+				ObjectMeta: v1.ObjectMeta{Name: "my-table"},
+				Spec: schemasv1alpha3.TableSpec{
+					Database: "my-database",
+					Name:     "my-table",
+					Schema: &schemasv1alpha3.TableSchema{
+						Postgres: &schemasv1alpha3.SQLTableSchema{
+							PrimaryKey: []string{"id"},
+							Columns: []*schemasv1alpha3.SQLTableColumn{
+								{
+									Name: "id",
+									Type: "text",
+									Constraints: &schemasv1alpha3.SQLTableColumnConstraints{
+										NotNull: new(bool),
+									},
+								},
+								{
+									Name: "name",
+									Type: "text",
+									Constraints: &schemasv1alpha3.SQLTableColumnConstraints{
+										NotNull: new(bool),
+									},
+								},
+							},
+						},
+					},
+				},
+				Status: schemasv1alpha3.TableStatus{},
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			test := test
+			t.Parallel()
+
+			actual, err := getApplyPod("", "", "", test.database, test.table)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			actualAnnotations := actual.ObjectMeta.Annotations
+
+			if !reflect.DeepEqual(actualAnnotations, test.expectedAnnotations) {
```

Comment:
can we use `assert.Equal(t, test.expectedAnnotations, actualAnnotations)` here, instead?